modules/stochastic_tools/include/userobjects/NeuralNetworkUserObject.h

```diff
@@ -0,0 +1,45 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+#include "GeneralUserObject.h"
+#include "MooseVariableInterface.h"
+
+// forward declarations
+class NeuralNetworkUserObject;
+
+template <>
+InputParameters validParams<GeneralUserObject>();
```

Comment:
This can be removed and all of methods and members of this class that are not overrides need documentation

Reply:
also remove unused forward declaraion